src/Components/v2/ArtworkFilter/Utils/urlFragmentFromState.tsx

```diff
@@ -0,0 +1,23 @@
+import qs from "qs"
+import { Filters } from "../ArtworkFilterContext"
+import { isDefaultFilter } from "./isDefaultFilter"
+
+/**
+ * Returns a string representing the query part of a URL. It removes default
+ * values, and rewrites keyword -> term.
+ */
+export const urlFragmentFromState = (state: Filters) => {
+  const { keyword: term } = state
+  const filters = Object.entries(state).reduce((acc, [key, value]) => {
+    if (isDefaultFilter(key, value) || key === "keyword") {
```

Comment:
More `term`/`keyword` stuff that we'll have to make work with apps other than Search.